apis/v1alpha1/tcproute_types.go

```diff
@@ -60,11 +60,28 @@ type TCPRouteStatus struct {
 
 // TCPRouteRule is the configuration for a given rule.
 type TCPRouteRule struct {
-	// Matches define conditions used for matching the rule against
-	// incoming TCP connections. Each match is independent, i.e. this
-	// rule will be matched if **any** one of the matches is satisfied.
-	// If unspecified, all requests from the associated gateway TCP
-	// listener will match.
+	// Matches define conditions used for matching the rule against incoming TCP
+	// connections. Each match is independent, i.e. this rule will be matched if
+	// **any** one of the matches is satisfied. If unspecified, all requests
```

Comment:
For some reason I'm finding the language a little harder to understand, maybe:

```
If unspecified (i.e. empty), this Rule will match all requests for the associated Listener.
```
```

Reply:
Updated, thanks!